Tests/TestingApps/PaywallsTester/Postprocessor.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
I was wondering whether it would make sense to write this in fastlane instead... But calling that as part of a scheme might be more complex, so I'm ok with this.